module/users/helper.js

```diff
@@ -273,9 +273,6 @@ module.exports = class UsersHelper {
           // Creates an array of promises based on users Input
           switch (type) {
             case constants.common.IMPROVEMENT_PROJECT:
```

Comment:
@ankitpws When this condition occurs we are not doing anything, So Is this case required?